File: jungyungee/Week1/영화감독숌.py
n = int(input())
i=1
j=0
if n <= 10000:
    num = str(i)
    while j < n:
        if "666" in num:
            j = j + 1
            if j == n:
                print(i)
                break
        i = i + 1
        num = str(i)

File: uoo6uoow/Week1/영화감독 숌.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        System.out.println(solution(N));
    }

    public static int solution(int N) {
        int answer = 0;
        ArrayList<Integer> sixList = new ArrayList<>();
        boolean contain; // 666이 들어있는지 확인하는 플래그
        int six;

        for (int i = 666; i < 10000000; i++) {
            String num = String.valueOf(i);
            contain = num.contains("666");

            if (contain) {
                sixList.add(i);
            }
        }

        answer = sixList.get(N - 1);
        return answer;
    }
}